### PR TITLE
 Provide guidance on validating OpenAPI specs

### LINKED ISSUE #125: Provide guidance on how to validate OpenAPI specs
Provide guidance on how to validate OpenAPI specs to make sure that generated Open API specs are valid.

### PR DESCRIPTION
**Pull request by tomkerkhove**
Closes #125

### CODE REVIEW

File: maturity-level-two/README.md [hunk 1]
```diff
@@ -1,2 +1,12 @@
 # Practical API Guidelines - Should have
 
+1. [Writing Open API Specifications](#writing-open-api-specifications)
+
+## Writing solid OpenAPI Specifications
```
Comment (tomkerkhove) [inline maturity-level-two/README.md]: I'm out of inspiration and welcome feedback for title + intro paragraph.

**Comment by tomkerkhove** [comment]
Thoughts on this @fgheysels @MassimoC ?

**Comment by fgheysels** [comment]
> Thoughts on this @fgheysels @MassimoC ?

Haven't taken the time yet to think about it.  Will do when I have some time to spare ...

**Comment by tomkerkhove** [comment]
No problem, thanks!

File: maturity-level-two/README.md (hunk 1 above)
Comment (MassimoC) [reply maturity-level-two/README.md]: Title is ok. We can add a point to the bullet list "unit test the open api document generation"

Comment (tomkerkhove) [reply maturity-level-two/README.md]: Updated, let me know if you like it.

**Pull request merged**